### PR TITLE
WIP : Implement the wrench

### PR DESCRIPTION
**Pull request by Phenix246**
- The wrench is able to rotate belt

### CODE REVIEW

File: src/main/java/net/qbar/common/init/QBarItems.java [hunk 1]
```diff
@@ -5,16 +5,21 @@
...
+import net.qbar.common.item.ItemWrench;
 
 @ObjectHolder(QBar.MODID)
 public class QBarItems
 {
     @ObjectHolder("punched_card")
     public static final ItemBase PUNCHED_CARD = null;
 
+    @ObjectHolder("wrench")
+    public static final ItemBase WRNECH       = null;
```
Comment (Ourten) [inline src/main/java/net/qbar/common/init/QBarItems.java]: Typo

File: src/main/java/net/qbar/common/item/ItemWrench.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    }
+
+    @Override
+    public EnumActionResult onItemUse(EntityPlayer player, World world, BlockPos pos, EnumHand hand, EnumFacing facing,
+            float hitX, float hitY, float hitZ)
+    {
+        TileEntity te = world.getTileEntity(pos);
+        if (!world.isRemote)
+        {
+            if (te instanceof TileBelt)
```
Comment (Ourten) [inline src/main/java/net/qbar/common/item/ItemWrench.java]: We should allow the wrench to rotate any block.
However when used on the up facing of a belt she need to switch it's slope state.

**Review by Ourten — CHANGES_REQUESTED** [review]

File: src/main/java/net/qbar/common/item/ItemWrench.java (hunk 2 above)
Comment (Phenix246) [reply src/main/java/net/qbar/common/item/ItemWrench.java]: Should we not use shift  for that ?

Comment (Ourten) [reply src/main/java/net/qbar/common/item/ItemWrench.java]: Yeah, it's better with it

**Comment by Phenix246** [comment]
Note : I broke the texture rendering in the last commit

**Pull request merged**